posts/google-summer-of-code-fellows/gsoc-2024-fellows.qmd

```diff
@@ -0,0 +1,294 @@
+In this blog, I will briefly describe what my project is and what I did as a part of it.
+
+
+> If you want to know more about me, you can connect with on [**LinkedIn**](https://www.linkedin.com/in/jay-landge-589439260/) and follow me on [**GitHub**](https://github.com/Jay-sanjay)
+
+
+
+# What is observational health research?
+
+Observational Health Research refers to studies that analyze real-world data (such as patient medical claims, electronic health records, etc.) to understand patient health. These studies often encompass a vast amount of data concerning patient care. An outstanding challenge here is that these datasets can become very complex and grow large enough to require advanced computing methods to process this information.
+
+
+
+# What is Patient Pathways ?
+
+Patient pathways refer to the journey that patients with specific medical conditions undergo in terms of their treatment. This concept goes beyond simple drug uptake statistics and looks at the sequence of treatments patients receive over time, including first-line treatments and subsequent therapies. Understanding patient pathways is essential for analyzing treatment patterns, adherence to clinical guidelines, and the rational use of drugs.
+To analyze patient pathways, one would typically use real-world data from sources such as electronic health records, claims data, and registries. However, barriers such as data interoperability and resource requirements have hindered the full utilization of real-world data for this purpose.
+
+So to address these challenges we wanna introduce to a set of tool to extract and analyze these patient pathways. These set of tool are based on the Observational Medical Outcomes Partnership (OMOP) common data model, which standardizes health care data to promote interoperability.
+
+
+# Project Description
+As part of this project with JuliaHealth, I developed a new package called [**OMOPCDMPathways.jl**](https://github.com/JuliaHealth/OMOPCDMPathways.jl). This package is designed for deployment in research projects, particularly those related to health and medical data analysis. This project takes much inspiration from [this](https://www.sciencedirect.com/science/article/pii/S016926072200462X?via%3Dihub) paper and explores the implementation of some of those ideas to develop new tools within the JuliaHealth Observational Health Subecosystem for exploring patient pathways. Additional new features and approaches were added and explored within this project. Additionally, I have authored a developer guide for the package, providing instructions on its use and contribution. This project provided me with hands-on experience in developing production-level code and exposed me to open-source software development practices. I had the opportunity to work in a team, under my mentor, and ensured the integration of the package with the rest of JuliaHealth, facilitating its adoption and usability within the community. 
```

Comment:
It would be great to actually cite the paper via this: https://quarto.org/docs/authoring/citations.html and to mention the name of the paper and authors in the post here.

Reply:
Can you also link the developer guide too?